### PR TITLE
Added conan packaging configuration.

### PR DESCRIPTION
**Pull request by dimi309**
As discussed. I have already published a [package of the master branch](https://bintray.com/dimi309/conan-packages/glm%3Ag-truc). I can also add the packaging config to the latest version, but let's take it step by step.

### CODE REVIEW

**Comment by Groovounet** [comment]
Thanks for contributing,
Christophe

**Comment by dimi309** [comment]
Thank you, for accepting my packaging things 😄  
Dimitri

File: util/conan-package/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from conans import ConanFile
+
+class GlmConan(ConanFile):
+    name = "glm"
+    version = "master"
+    generators = "txt"
+    url="https://github.com/g-truc/glm"
+    description="OpenGL Mathematics (GLM)"
+    license = "https://github.com/g-truc/glm/blob/manual/copying.txt"
+    exports = ["FindGLM.cmake", "lib_licenses/*", os.sep.join([".", "..", "..", "*"])]
```
Comment (memsharded) [inline util/conan-package/conanfile.py]: You might take advantage of the ``exports_sources`` functionality to store the sources and the .cmake file. I would only keep the license as ``exports``. It is much more efficient to use ``exports_sources`` for source code.

File: util/conan-package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    def build(self):
+        self.output.warn("No compilation necessary for GLM")
+        self.output.warn(os.sep.join([".", "..", "..", "*"]))
```
Comment (memsharded) [inline util/conan-package/conanfile.py]: What is the purpose of the second warning? I would remove it, and maybe change the first one for a ``output.info()`` message, as it is not something that should concern the user.

File: util/conan-package/test_package/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake
+import os
+
+channel = os.getenv("CONAN_CHANNEL", "testing")
```
Comment (memsharded) [inline util/conan-package/test_package/conanfile.py]: From conan 0.25 you can simplify this test_package recipe, removing the user/channel, and also the ``requires``, so it is not necessary to change them or change versions when testing a different one. Then, use ``conan create user/channel`` better.

Comment (dimi309) [inline util/conan-package/test_package/conanfile.py]: Thanks for the feedback! I have implemented the changes and introduced a pull request: https://github.com/g-truc/glm/pull/662

**Pull request merged**